gulpfile.js

```diff
@@ -425,10 +425,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.1.1'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.13.2'))
+    .pipe(replace('[ios.current-app-version]', '2.14.1'))
     .pipe(replace('[android.latest-os-version]', '12'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.13.3'))
+    .pipe(replace('[android.current-app-version]', '2.14.0'))
```

Comment:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.14.1'))
```

See https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.14.1-RC1